cirq-core/cirq/linalg/combinators.py

```diff
@@ -39,7 +39,9 @@ def kron(*factors: Union[np.ndarray, complex, float], shape_len: int = 2) -> np.
     Returns:
         The kronecker product of all the inputs.
     """
-    product = np.ones(shape=(1,) * shape_len)
+    product: np.ndarray[
```

Comment:
I think this can just be np.ndarray (worked locally for me).  The extra typing likely is just more confusing than helpful.

Reply:
Done.